spark-etl/src/main/scala/geotrellis/spark/etl/config/Backend.scala

```diff
@@ -0,0 +1,3 @@
+package geotrellis.spark.etl.config
+
+case class Backend(`type`: BackendType, path: String, profile: Option[String] = None)
```

Comment:
This really begs for `profile` to be an `Option[BackendProfile]`, see comment in the parser for details.